src/AspNet.Security.OAuth.Gitter/AspNet.Security.OAuth.Gitter.csproj

```diff
@@ -3,21 +3,25 @@
   <Import Project="..\..\build\packages.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net451;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
   </PropertyGroup>
 
   <PropertyGroup>
     <Description>ASP.NET Core security middleware enabling Gitter authentication.</Description>
-    <Authors>Maxime Roussin-Bélanger</Authors>
+    <Authors>Hayden Hancock</Authors>
```

Comment:
Please don't remove the name of the initial contributor. You can add yours while preserving the initial one'. E.g:

<Authors>Maxime Roussin-Bélanger;Hayden Hancock</Authors>